RestAPI.cs

```diff
@@ -43,12 +43,12 @@ public class RestAPI
             if (string.IsNullOrEmpty(url))
                 throw new Exception("Please use a valid WooCommerce Restful API url.");
 
-            url = url.Trim().ToLower().TrimEnd('/');

```

Comment:
At least you can fix it by now, instead of waiting for the new version. The library will be updated.